graph/ann_validation.go

```diff
@@ -48,70 +45,3 @@ func ValidateNodeAnn(a *lnwire.NodeAnnouncement) error {
 
 	return nil
 }
-
```

Comment:
same thread as before: should we instead just extend these functions with switches on the types instead of moving the validation to the message interfaces?